L9/web/web.py

```diff
@@ -0,0 +1,41 @@
+import streamlit as st
+import pandas as pd
+from scipy import spatial
+from sentence_transformers import SentenceTransformer
+
+st.title('Online shop')
+
+DATA_URL = 'https://gist.github.com/ko3a4ok/9e8128ca2917a2b9379a716b50ef621c/raw/0df0af04199f0cfcfe6fb7531f467bbbaf0e183e/cymbal_product_desc.txt'
+
+
+@st.cache(allow_output_mutation=True)
+def get_model():
+  return SentenceTransformer('TaylorAI/bge-micro')
+
+
+data_load_state = st.text('Loading data...')
+
+
+@st.cache(allow_output_mutation=True)
+def load_data():
+  df = pd.read_fwf(DATA_URL).iloc[:1000, :1]
```

Comment:
remove hardcode

Reply:
the data file is too big. It would take over 15 minutes to generate embedding for the whole content